app/http.php

```diff
@@ -51,7 +51,7 @@
     Console::success('Reload completed...');
 });
 
-Files::load(__DIR__ . '/../public');
+Files::load(__DIR__ . '/../console');
```

Comment:
In theory, I think we can load both folders.

Reply:
we don't need the public anymore 👍🏻